### DISCUSSION TITLE
train keypoints estimation using pseudo labelling with a custom dataset

### DISCUSSION THREAD

**Discussion by tracyqian0720** [facebookresearch/detectron2#3252] [Q&A]
Hi , I am looking to train the keypoints estimation model using pseudo labelling with a custom dataset that I collected myself. Only the keypoints are desired and that's what is in my json file : names of key points and the coordinates (obtained using outputs["instances"].pred_keypoints.cpu().detach().numpy() ), 

I am struggling to train the keypoint model cuz I have never worked with COCO format before but it seems like it is the format that is required for detectron, how can I convert my json data to COCO format? or should I generate a new set of data since I didn't include bbox and other information? Or they wouldn't matter if they are not desired as an output? 

Thank you so much !
